erpnext/stock/doctype/stock_entry/test_stock_entry.py

```
@@ -2280,6 +2280,98 @@ def test_manufacture_entry_without_wo(self):
+				assert i.amount == 100 * (OVERRIDE_AMOUNT / 4)
+			if i.item_code == rm_item3:
+				assert i.qty == (QTY_TO_MADE / 4)
+				assert i.transfer_qty == (QTY_TO_MADE / 4)
+				assert i.amount == 20 * (QTY_TO_MADE / 4)
+			if i.item_code == rm_item4:
+				rm_item4_in_se = True
+				assert i.qty == QTY_TO_MADE / 4
+				assert i.transfer_qty == QTY_TO_MADE / 4
+				assert i.amount == 20 * (QTY_TO_MADE / 4)
+
+		assert rm_item4_in_se == True
```

Ruff (0.15.0)

[error] Avoid equality comparisons to `True`; use `rm_item4_in_se:` for truth checks

Replace with `rm_item4_in_se`

(E712)